src/routes/spaced_repetition.py

```
@@ -372,3 +372,99 @@ def import_pipeline_cards():
+                .all()
+            )
+
+        cards_data = [c.to_dict() for c in due_cards]
+
+        return jsonify({
+            'status': 'success',
+            'cards': cards_data,
+            'count': len(cards_data)
+        }), 200
+
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)